src/main/app/src/app/shared/progress-snackbar/progress-snackbar.component.less

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely
```

Comment:
new file?

```suggestion
 * SPDX-FileCopyrightText: 2024 Lifely
```

Reply:
Same here; This is code from ATOS taken out of util.service.ts; so trivial I guess.

Reply:
Same here; This is a new component but the code was ATOS and moved out of util.service.ts

Reply:
ok, clear! sorry about the confusion. :-)

Reply:
No worries; was trivial so would have been better for me to add some explanation